test/system/src/main/java/io/pravega/test/system/framework/Utils.java

```diff
@@ -116,10 +119,36 @@ public static Service createPravegaSegmentStoreService(final URI zkUri, final UR
                 return  new PravegaSegmentStoreDockerService(serviceId, zkUri, hdfsUri, contUri);
             case KUBERNETES:
             default:
-                return new PravegaSegmentStoreK8sService(serviceId, zkUri);
+                return new PravegaSegmentStoreK8sService(serviceId, zkUri, getPravegaProperties());
+        }
+    }
+
+    public static PravegaProperties getPravegaProperties() {
+        if (!SECURITY_ENABLED) {
+            return PravegaProperties.builder().build();
+        } else {
+            return PravegaProperties.builder().buildWithSecurity();
         }
     }
 
+    public static ClientConfig getClientConfig(URI controllerUri) {
```

Comment:
`get...()` seems to give an impression that the returned object is pre-created/cached and that it is inexpensive to call it again and again. In this case, the code is preparing/building an object upon call, so it might make more sense to call it `build..()/prepare...()`.

Reply:
Ok. will change.

Reply:
done